screens/LuresResults.js

```diff
@@ -9,89 +9,1035 @@ import {
+    "image": "https://www.sail.ca/media/catalog/product/d/i/direct-lt8-yb_yellow---black.jpg",
+    "color1": "gold",
+    "color2": "grey",
+    "size": "8",
+    "weight": "0",
+    "brand": "LENTHOM",
+    "model": "Original Spoon - 0.25",
+    "type": "spoon",
+    "price": 4.99
+    },
+    {
+    "image": "https://www.sail.ca/media/catalog/product/n/o/normark-bmsh09_smelt__01.jpg",
+    "color1": "grey",
+    "color2": "green",
+    "size": "9",
+    "weight": "0",
+    "brand": "STORM",
+    "model": "Boom Shad Soft Swimbait - 9",
+    "type": "swimbait",
+    "price": 6.99
+    },
+    {
+    "image": "https://www.sail.ca/media/catalog/product/m/u/musky-101_lper.jpg",
+    "color1": "brown",
+    "color2": "grey",
+    "size": "9",
+    "weight": "4",
+    "brand": "MUSKY INNOVATION",
+    "model": "Regular Bull Dawg Crankbait - 9",
+    "type": "crankbait",
+    "price": 34.99
+    },
+    {
+    "image": "https://www.sail.ca/media/catalog/product/m/u/musky-101_lper.jpg",
+    "color1": "brown",
+    "color2": "grey",
+    "size": "9",
+    "weight": "4",
+    "brand": "MUSKY INNOVATION",
+    "model": "Regular Bull Dawg Crankbait - 9",
+    "type": "crankbait",
+    "price": 34.99
+    }
+    ])
 
-  const renderItem = ({ item }) => (
-    <View
-      style={[
-        flex_style.flexColumn,
-        padding_styles.space_md,
-        flex_style.width100,
-        btn_style.buttonFullColor,
-      ]}
-    >
-      <Image
-        source={{ uri: item.image }}
-        style={[img_styles.square_image_md, padding_styles.space_md]}
-      />
-      <Text
-        style={[
-          text_style.md,
-          text_style.fontColorBlack,
-          padding_styles.space_s,
-        ]}
-      >
-        {loadTranslations("type")}: {item.type}
-      </Text>
-      <Text
-        style={[
-          text_style.md,
-          text_style.fontColorBlack,
-          padding_styles.space_s,
-        ]}
-      >
-        {loadTranslations("color")}: {item.color}
-      </Text>
-      <Text
-        style={[
-          text_style.md,
-          text_style.fontColorBlack,
-          padding_styles.space_s,
-        ]}
-      >
-        {loadTranslations("weight")}: {item.weight}
-      </Text>
-      <Text
-        style={[
-          text_style.md,
-          text_style.fontColorBlack,
-          padding_styles.space_s,
-        ]}
-      >
-        {loadTranslations("effectivenessPercentage")}: {item.effectiveness}%
-      </Text>
+  const moonPhaseLabels = {
+    1: 'New Moon',
+    2: 'First Quarter',
+    3: 'Full Moon',
+    4: 'Last Quarter'
+  };
+
+  const WeatherAndMoonPhase = ({ weather, moonPhase }) => (
+    <View style={{ flexDirection: 'row', justifyContent: 'space-between', padding: 10, alignItems: 'center' }}>
+      <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+      <MaterialIcons name="dark-mode" size={22} color="black" />
+      <Text style={{ marginLeft: 5 }}>{moonPhaseLabels[moonPhase.prevPhase.phase]}</Text>
```

Comment:
so we need to display both moon phases
and if u look at the issue the way it works is they need to know if they're approaching a certain type of moon phase because some produce better fishing results than others